.github/workflows/autopublish.yml

```diff
@@ -2,10 +2,10 @@ name: Publish website
 on:
   push:
     branches:
-      - master
+        - master
```

Comment:
is this intentional?

Reply:
It's not – don't know what went wrong here.